docker-compose.yml

```diff
@@ -34,13 +34,45 @@ services:
     ports:
       - "26381:26381"
 
+  cluster_master_1:
+    build: docker/cluster/master_1
+    ports:
+      - "7000:7000"
```

Comment:
Since "regular" redis master/slave claimed 6379/6380, and sentinel is standardized on 26379/26380+, let's keep this in theme and use *either* 46379+ or 16379+.

Reply:
Changed to 16379+